rules/dns-what-and-how-it-works/rule.md

```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you know DNS, what it is and how it works?
+uri: dns-what-and-how-it-works
+authors:
+  - title: Dhruv Mathur
+    url: https://www.ssw.com.au/people/dhruv-mathur/
+created: 2024-02-28T07:04:54.685Z
+guid: 21275f4c-aaf4-4964-9d25-804f3cb56e75
+---
+
+            
+The Domain Name System (DNS) is akin to the internet's phonebook. It's easy to remember a website's name, like www.ssw.com.au, but computers and networks need numerical IP addresses to access websites. DNS translates these user-friendly domain names to IP addresses, allowing browsers to load internet resources.
+
+### DNS explained
+
+`youtube: https://www.youtube.com/embed/{VIDEO_ID}`
+
+**Video: Everything You Need to Know About DNS**
```

Comment:
Hi @Dhruv-0987 

- [ ] 1. Please add the video lenght at the end as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

Reply:
✅ Done - added video length at the end.